src/app/src/constants.js

```diff
@@ -15,3 +15,8 @@ export const DEFAULT_BASEMAP_URL =
     'https://server.arcgisonline.com/ArcGIS/rest/services/Canvas/World_Light_Gray_Base/MapServer/tile/{z}/{y}/{x}.png';
 export const SATELLITE_BASEMAP_URL =
     'https://services.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}.png';
+
+export const NC_WEST = -84.57275390625;
```

Comment:
Consider adding a comment indicating the source of these numbers

Reply:
I actually just zoomed out to where it seemed like all of North Carolina was in the window and then took the bounds from the leaflet map. Is there a more precise set of values we want to use?

Reply:
Let's use the ones from here and reference them: https://observablehq.com/@rdmurphy/u-s-state-bounding-boxes. The source of the data is: https://github.com/topojson/us-atlas#states-10m.json.

Reply:
1002636848c13eeee3336e4e24e1ff56e2026aed